spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractMessageChannelBinder.java

```diff
@@ -824,5 +844,12 @@ ErrorInfrastructure getErrorInfrastructure() {
 		}
 
 	}
+	
+	@SuppressWarnings("serial")
+	public class BindingCreatedEvent extends ApplicationEvent {
```

Comment:
Great! But can we make this as a top-level class?
And may be extract a super class like `StreamEvent`?
Similar to what we have in SI with the `IntegrationEvent`

Reply:
Top level yes, good point
As far as adding `super`. . . not ready for it yet, will revisit for 2.1